src/app/calendar/parcelCalendarFunctions.ts

```diff
@@ -1,44 +1,33 @@
 import supabase, { Schema } from "@/supabase";
 import { CalendarEvent } from "@/components/Calendar/Calendar";
 
-const COLLECTION_DURATION_MS = 30 * 60 * 1000;
+export type ParcelWithClientName = Schema["parcels"] & { clients: { full_name: string } | null };
 
 export interface LocationColorMap {
     [location: string]: { color: string; text: string };
 }
 
-export interface ClientMap {
-    [primary_key: string]: Schema["clients"];
-}
-
-export const getClientMap = async (): Promise<ClientMap> => {
-    const response = await supabase.from("clients").select();
-    const clients = response.data ?? [];
-    const clientMap: ClientMap = {};
+const COLLECTION_DURATION_MS = 30 * 60 * 1000;
```

Comment:
this has moved again! can you put it back at the top of the file :)

Reply:
Done